ci/migrate.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openmeterio/openmeter/ci/internal/dagger"
+	"github.com/sourcegraph/conc/pool"
+)
+
+func (m *Ci) Migrate() *Migrate {
+	return &Migrate{
+		Source: m.Source,
+	}
+}
+
+type Migrate struct {
+	Source *dagger.Directory
+}
+
+func (m *Migrate) Check(
+	ctx context.Context,
+) error {
```

Comment:
```suggestion
func (m *Migrate) Check(ctx context.Context) error {
```

Reply:
I do not think the original foramt makes the code more readable. Quite the opposite in my opinion.